src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -124,6 +124,10 @@ public SwerveModulePosition[] getModulePositions() {
       rearRight.getPosition()
     };
   }
+
+  public MatchState getMatchState() {
+    return this.matchState;
+  }
   ;
```

Comment:
Why do we have this? I don't think it's used anywhere, and IMO the main source for the matchState should be `RobotContainer`, not here

Reply:
removed for now